pollux/lib/core/src/main/scala/io/iohk/atala/pollux/core/service/verification/VcVerificationServiceError.scala

```diff
@@ -0,0 +1,9 @@
+package io.iohk.atala.pollux.core.service.verification
+
+sealed trait VcVerificationServiceError {
+  def error: String
+}
```

Comment:
Would be nice if we can apply error handling ADR to new code https://github.com/hyperledger-labs/open-enterprise-agent/blob/eb6b6493dfda14764b2675f0b2ffda0e826d897c/docs/decisions/20240116-use-zio-failures-and-defects-effectively.md#option-2---practical-implementation

Reply:
Will do in a follow up